src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,168 @@
+# Eye-tracking including gaze position and pupil size
+
+Support for eye-tracking dataset developed as a
+[BIDS Extension Proposal](../extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../introduction.md#citing-bids) on how to
+appropriately credit this extension when referring to it in the context
+of the academic literature.
+
+## Terminology and conventions
+
+Eyetracking-BIDS is fully consistent with the BIDS specification as a whole.
+However, BIDS was initially developed in the context of MRI,
+so some terminology may be unfamiliar to researchers from other fields.
+This section adds clarifications to
+[Common Principles - Definitions](../common-principles.md#definitions) for the
+eye-tracking context.
+
+-   **Eye-tracker** - refers to the apparatus allowing the records of gaze
+    position and/or pupil size.
+
+-   **Screen** - Video display used to present visual stimulation (for example
+    projector, monitor, tablet).
+
+## Eye-tracking data
+
+With EyeTracking-BIDS, we wish to promote the adoption of good practices in
+the management of scientific data.
+
+### Template:
+
+<!--
+This block generates a filetree exanple.
+A guide for editing it can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md.
+-->
+{{ MACROS___make_filetree_example(
+
+   {
+   "sub-<label>": {
+      "<datatype>": {
+         "<matches>_eyetrack.json": "",
+         "<matches>_eyetrack.tsv.gz": "",
+         },
+      }
+   }
+
+) }}
+
+Eye-tracking data MUST be stored in the main data recording modality or
+ `<datatype>` directory (for example `func` when combined with fMRI, or
+ `beh` when combined with behavioral measures).
+
+In the template filenames, the `<matches>` part corresponds to task filename
+before the suffix. For example if fMRI is considered the main modality with
+files such as `sub-01_ses-1_task-pRF_run-1_bold.nii.gz`, `<matches>` would
+then correspond to `sub-01_ses-1_task-pRF_run-1`.
```

Comment:
I think this oversimplifies it. For example, you won't include `part` or `echo`.

Reply:
So what do you suggest ? to add part and echo ?

Reply:
Maybe the "easiest" way to deal with this is to:

1. add the eyetrack suffix to `task.yml` rule file about time series (where we keep info for stim and physio entites)

https://github.com/bids-standard/bids-specification/blob/cba4ebcaa039cd10d6dcec46b53110c4024f924d/src/schema/rules/files/raw/task.yaml#L24

2. then add the 'eyetrack' suffix to the macro calls that is in charge of creating the filename templates for each datatype that will support eyetracking

https://github.com/bids-standard/bids-specification/blob/cba4ebcaa039cd10d6dcec46b53110c4024f924d/src/modality-specific-files/electroencephalography.md?plain=1#L25

Then we would not have

Reply:
https://github.com/bids-standard/bids-specification/pull/1128/commits/f0635239be55013903955381a147179521066a20

Please have a look at that... I'm sure it works everywhere... if you feel it is good please Resolve the conversation...

Reply:
Worked as expected.

Thanks